lib/TH/generic/THVectorDefault.c

```diff
@@ -17,68 +32,100 @@ void THVector_(fill_DEFAULT)(real *x, const real c, const ptrdiff_t n) {
     x[i] = c;
 }
 
+void THVector_(cadd_DEFAULT)(real *z, const real *x, const real *y, const real c, const ptrdiff_t n)
```

Comment:
I think we should get rid of the hand-unrolling, but I'm okay saving it for a future PR.